pages/parent/signup.tsx

```diff
@@ -421,55 +401,17 @@ export default function ParticipantInfo({
             body: JSON.stringify(userData),
```

Comment:
Here, I think we want userData to use the emergency contact, since I don't actually think the username name columns get filled out initially when parents signup for an account, this is actually where they set their name. 

```
const userData = {
    firstName: emergFirstName,
    lastName: emergLastName,
    role: roles.PARENT,
    roleData: parentData,
};
```

My understanding is that we don't actually need these lines you had earlier:
```
    // Parent info
    const [parentFirstName, setParentFirstName] = useLocalStorage(
        "parentFirstName",
        "",
    );
    const [parentLastName, setParentLastName] = useLocalStorage(
        "parentLastName",
        "",
    );
    const [parentPhoneNumber, setParentPhoneNumber] = useLocalStorage(
        "parentPhoneNumber",
        "",
    );
    const [parentRelationship, setParentRelationship] = useLocalStorage(
        "parentRelationship",
        "",
    );
```